index.d.ts

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+
+type DateInfo = {
+    day: number;
+    month: number;
+    year: number;
+}
+
+type Range = {
+    from: DateInfo | null;
+    To: DateInfo | null;

```

Comment:
Let's use `undefined` instead of `null`.

Reply:
I took null from docs:

![image](https://user-images.githubusercontent.com/35793921/68698059-59053180-0595-11ea-9888-1a4e4b39e0c5.png)